src/main/resources/assets/js/components/AppComponent.jsx

```diff
@@ -34,9 +72,14 @@ define([
           <td className="text-right">{model.get("mem")}</td>
           <td className="text-right">{model.get("cpus")}</td>
           <td className="text-right">
-            <span className={runningTasksClassSet}>
-              {model.formatTasksRunning()}
-            </span> / {model.get("instances")}
+            <span className={runningTasksClassSet}>{model.formatTasksRunning()}</span>
+            <span> / {model.get("instances")}</span>
+              {
```

Comment:
Let's move logic up above the view definition. Create a variable and assign it to null or `AppHealthBarComponent` or what it is going to be called